src/cpmBucketManager.js

```diff
@@ -118,17 +119,35 @@ function getCpmTarget(cpm, bucket, granularityMultiplier) {
   const precision = typeof bucket.precision !== 'undefined' ? bucket.precision : _defaultPrecision;
```

Comment:
This change LGTM, but isn't the use case for this already covered by this configurable `precision` (as explained [here](https://docs.prebid.org/dev-docs/publisher-api-reference/setConfig.html#customCPMObject))? This is probably more about documentation, it's not clear to me why I'd want to use one over the other.

Reply:
Precision states where the rounding will occur, but the rounding is always down. This change allows for other rounding methods. 

e.g. 
Current
cpm = 2.1456, increment = .05, precision = 3
precise cpm = 2.145, rounded down to nearest increment = 2.10
Final result = 2.100

With alternate rounding
cpm = 2.1456, increment = 0.5 precision = 4
precise cpm = 2.1456, rounded with Math.round to nearest increment = 2.15
Final result = 2.1500

At least, that's how I understand it. As far as I can tell, the precision determines what is being rounded down, but it is always rounded down, and always to the nearest increment, which in effect just tacks extra zeroes to the end of the result.

Then again, I'm mostly basing that off the unit tests, which aren't super comprehensive for precision, and the documentation, which only shows the default of 2. If precision is supposed to affect rounding in any way, I don't get it. As long as the precision is greater than or equal to the number of decimal places in the increment, rounding will be unaffected by precision, which makes this change useful.

Reply:
You're right, I thought you could get the equivalent of `Math.round` by increasing the precision by 1, but that's not exactly what happens.